docs/using/docker-image.rst

```diff
@@ -29,7 +29,7 @@ world tutorial-app, let's call it "helloworld":
 
 ::
 
-    $ tsuru app create helloworld static
+    $ tsuru app create helloworld static --team
```

Comment:
this is not accurate, if the user runs this exact command tsuru will ask the user to specify a valid team